src/app/app.component.html

```diff
@@ -4,8 +4,25 @@
 </mo-header>
 
 <ng-container *ngIf="weatherData.length==locLength">
-  <ng-container *ngFor="let data of weatherData; index as i">
-    <button (click)="locationID = i">{{data.location.name}}</button>
-  </ng-container>
-  <app-weather-data *ngIf="locationID || locationID==0" [data]="weatherData[locationID]"></app-weather-data>
+  <div id="buttonDiv">
+    <ng-container *ngFor="let data of weatherData; index as i">
+      <button (click)="locationID = i"><b>{{data.location.name}}</b></button>
+    </ng-container>
+  </div>
+
+  <div id="smallSelect">
```

Comment:
you can get away with it here as it's the root component, but we generally avoid IDs, as with everything being a component, you could end up with multiple of the same ID on a page, if the component gets rendered out multiple times
So generally stick with classes as Angular will scope them to the component (it may do this to IDs as well, I'm not sure! 🤷‍♂️ but classes generally seem more appropriate to me in the angular world 👍 )